x/did/client/rest/tx.go

```diff
@@ -12,38 +14,30 @@ import (
 	"github.com/ixofoundation/ixo-blockchain/x/did/internal/keeper"
 	"github.com/ixofoundation/ixo-blockchain/x/did/internal/types"
 	"github.com/ixofoundation/ixo-blockchain/x/ixo"
+
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
 )
 
 func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
-	r.HandleFunc("/did", createDidRequestHandler(cliCtx)).Methods("POST")
-	r.HandleFunc("/credential", addCredentialRequestHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/did/create_did", createDidRequestHandler(cliCtx)).Methods("POST")
```

Comment:
Sorry, actually please change to match the message name. So: `/did/add_did`

Please also change `createDidRequestHandler` to `addDidRequestHandler`